src/Draco.Editor.Web/app/logo-handler.js

```diff
@@ -0,0 +1,48 @@
+import {get} from 'https';
+
+export function CreateThemeBasedLogo() {
+    var logoLight = "";
+    get("https://raw.githubusercontent.com/Draco-lang/Language-suggestions/main/Resources/Logo-Short.svg", (response) => logoLight = response);
```

Comment:
I think this is bugged, the callback  can complete after exiting this function.
Use async/await for that